pyblish_qml/models.py

```diff
@@ -453,7 +454,7 @@ def update_with_result(self, result):
             item.isProcessing = False
             item.currentProgress = 1
             item.processed = True
-            item.hasWarning = any([
+            item.hasWarning = item.persistWarning or any([
```

Comment:
Could this be:

```python
item.hasWarning = item.hasWarning or any([...
```

That way, it'll be set once, and then re-used. That way we'd save on introducing an additional variable too.

Reply:
Oh, yes ! That line can change to
```python
item.hasWarning |= any([...
```

Reply:
Done, is this what you were saying ?

Reply:
Oh, wait, yes, I might not need that `persistWarning `

Reply:
Done ! Awesome !

Reply:
Actually, I would prefer `item.hasWarning = item.hasWarning or ...`

Because, I'm not sure if you know, `|=` does not mean "or equals". It means "bitwise or equals" and I'm really rusty on my bitwise math. I'm not sure you knew `True |= False == True` which it does in this case and should, but to me that's black magic and I will be looking at that line sometime in the future wondering "What does that do?".

It's not as pretty as what you've got, but I think it will save someone time in the future. Does that make sense?

Reply:
> I'm not sure if you know, |= does not mean "or equals".

I am not aware that !
Sure, of course.

Reply:
It's in fix 4 now.